docs/design/path-computation.md

```diff
@@ -28,12 +18,15 @@ The pce project has multiple unit tests that excercise most / all of the core lo
 
 Here are the basic steps to getting up and running:
 
-1. Ensure you can run this maven project, run the tests, and they all should pass.
-2. Ensure you can run the acceptance tests:
-    1. make atdd tags="@PCE,@FPATH"
+1. Ensure you can run kilda-pce maven project, run the tests, and they all should pass.
 
-## Notea
+## Path computation strategies
 
-In addition to this sparse guide, there are more notes in PathComputationTests.java regarding how the acceptance is executed.  That should give you a good guide to how the code comes together.
+Strategy for each path computation should be provided explicitly. If flow
+create/update/reroute request doesn't has path strategy parameter then default 
```

Comment:
reroute should use flow specified param, please separate reroute from create/update

Reply:
to make it clear by the moment of reroute some strategy will be already stored in flow itself, so we just reuse it

Reply:
Reworked doc.